azurecompute/src/main/java/org/jclouds/azurecompute/domain/OSVirtualHardDisk.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.azurecompute.domain;
+
+import com.google.auto.value.AutoValue;
+import org.jclouds.javax.annotation.Nullable;
+
+import java.net.URI;
+
+/**
+ * @see <a href="http://msdn.microsoft.com/en-us/library/azure/jj157186.aspx#OSVirtualHardDisk" >api</a>
+ */
+@AutoValue
+public abstract class OSVirtualHardDisk {
+
+   /**
+    * Specifies the caching mode of the operating system disk.
+    * This setting impacts the consistency and performance of the disk.
+    * Possible values are:
+    * ReadOnly
+    * ReadWrite
+    * The default value is ReadWrite
+    */
+   @Nullable public abstract String hostCaching();
+
```

Comment:
Looks like also an optional `DiskLabel` Specifies the description of the disk. can be specified. Can you add it?